airbyte-webapp/.husky/pre-commit

```diff
@@ -0,0 +1 @@
+cd airbyte-webapp && npx --no lint-staged
```

Comment:
From the #16984, no sure if it matters, but this is what my husky migration script generated.
```
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"
```

Reply:
This should not be needed since I followed the manual update guide, and that `husky.sh` script does not exist this way.